<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="152148" started-at="2021-03-19T07:52:53Z" finished-at="2021-03-19T07:55:25Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="152148" started-at="2021-03-19T07:52:53Z" finished-at="2021-03-19T07:55:25Z">
      <class name="jl.project.pa11yTest.UserRequirement5_Pa11yTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:jl.project.pa11yTest.UserRequirement5_Pa11yTest@1f3f4916]" name="setup" is-config="true" duration-ms="152041" started-at="2021-03-19T08:52:53Z" finished-at="2021-03-19T08:55:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="pa11yTest()[pri:0, instance:jl.project.pa11yTest.UserRequirement5_Pa11yTest@1f3f4916]" name="pa11yTest" duration-ms="28" started-at="2021-03-19T08:55:25Z" finished-at="2021-03-19T08:55:25Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionErrorat org.junit.Assert.fail(Assert.java:87)at org.junit.Assert.assertTrue(Assert.java:42)at org.junit.Assert.assertTrue(Assert.java:53)at jl.project.pa11yTest.UserRequirement5_Pa11yTest.pa11yTest(UserRequirement5_Pa11yTest.java:158)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pa11yTest -->
        <test-method status="PASS" signature="release()[pri:0, instance:jl.project.pa11yTest.UserRequirement5_Pa11yTest@1f3f4916]" name="release" is-config="true" duration-ms="7" started-at="2021-03-19T08:55:25Z" finished-at="2021-03-19T08:55:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- release -->
      </class> <!-- jl.project.pa11yTest.UserRequirement5_Pa11yTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
